backend/authentication/models.py

```diff
@@ -55,6 +57,8 @@ class User(AbstractUser):
     fav_sport_2 = models.TextField(default="")
     fav_sport_3 = models.TextField(default="")
 
+    badges = ArrayField(models.CharField(max_length=30), default=empty_list)
```

Comment:
Should we have the information of which badge is given for which event in User table? If we want to show it on User profiles, the table will require to keep BadgeRecord instead of just badge names.

Reply:
I do not think we are required to show the event information of individual badges. Users might have many badges that can cover the page. I am planning to show the counts of the badges in the user profile.

Reply:
I think Berkay's comment answers the concern